docs/configuration.rst

```diff
@@ -438,6 +438,39 @@ Default: ``[]``
 
 Types: ``["array"]``
 
+.. _configuration_report_template_paths:
+
+report.template_paths
+~~~~~~~~~~~~~~~~~~~~~
+
+List of paths to load templates from
+
+Default: ``["\/home\/daniel\/www\/phpbench\/phpbench\/lib\/Extension\/..\/..\/templates"]``
+
+Types: ``["array"]``
+
+.. _configuration_report_html_output_dir:
+
+report.html_output_dir
+~~~~~~~~~~~~~~~~~~~~~~
+
+Path in which to render HTML reports
```

Comment:
This is either a global default, or a per-report setting - or both. This could be overridden at the report level whilst providing a default output dir.